php/class-terminus.php

```diff
@@ -162,15 +162,37 @@ static function prompt($message = '', $params = array(), $default=null) {
   }
 
   /**
-   * Prompt the user for input
+   * Gets input from STDIN silently
+   * By: Troels Knak-Nielsen
+   * From: http://www.sitepoint.com/interactive-cli-password-prompt-in-php/
    *
-   * @param string $message
+   * @param [string] $message Message to give at prompt
+   * @param [array]  $params  Params given to command
+   * @param [string] $default Default value
+   * @return [string] $message
    */
-  static function promptSecret($message = '', $params = array(), $default=null) {
-    exec("stty -echo");
-    $response = Terminus::prompt($message, $params);
-    exec("stty echo");
-    Terminus::line();
+  static function promptSecret($message = '', $params = array(), $default = null) {
+    if (preg_match('/^win/i', PHP_OS)) {
```

Comment:
There is a util function for this, though I don't have a preference as to whether we want to use it or not: https://github.com/pantheon-systems/cli/blob/cd447e88f183107d2a991bdd708abcd34ae5fdfe/php/utils.php#L208